### PR TITLE
Adding a Task for jib-maven

### PR DESCRIPTION
**Pull request by kimsterv**
# Changes

Adds a Task for jib-maven

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/catalog/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

File: jib-maven/README.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+```
+apiVersion: tekton.dev/v1alpha1
+kind: TaskRun
+metadata:
+  name: example-jib-maven
+spec:
+  taskRef:
+    name: jib-maven
+  inputs:
```
Comment (dlorenc) [inline jib-maven/README.md]: Should this produce an image output resource? I think you can change the input parameter to an output resource.

File: jib-maven/jib-maven.yaml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+      subPath: jib-cache
+
+  volumes:
+  - name: empty-dir-volume
+    emptyDir: {}
+
+---
+
+apiVersion: tekton.dev/v1alpha1
+kind: TaskRun
```
Comment (dlorenc) [inline jib-maven/jib-maven.yaml]: Can you move the task run to a new file?

File: jib-maven/README.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+```
+kubectl apply -f https://raw.githubusercontent.com/tektoncd/catalog/master/jib-maven/jib-maven.yaml
+```
+
+## Inputs
+
+### Parameters
+
+- **IMAGE**: The Docker image name to apply to the newly built image. (*required*)
```
Comment (dlorenc) [inline jib-maven/README.md]: Now you just have to fix up the docs and this is good to go!

File: jib-maven/jib-maven.yaml (hunk 2 above)
Comment (ImJasonH) [reply jib-maven/jib-maven.yaml]: I think we should just remove this task run section, that's what I did for the other Tasks. The example task is in the README, where users hopefully won't be confused when they apply all YAMLs in the directory.

**Pull request merged**